addons/marketing_campaign/marketing_campaign.py

```diff
@@ -400,8 +400,6 @@ class marketing_campaign_activity(osv.osv):
   """),
         'email_template_id': fields.many2one('mail.template', "Email Template", help='The email to send when this activity is activated'),
         'report_id': fields.many2one('ir.actions.report.xml', "Report", help='The report to generate when this activity is activated', ),
-        'report_directory_id': fields.many2one('document.directory','Directory',
-                                help="This folder is used to store the generated reports"),
```

Comment:
i think we should keep this field and document dependency. you should remove dependencies on document which are due to index.